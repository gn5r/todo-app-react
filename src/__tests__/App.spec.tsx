import "@testing-library/jest-dom/vitest";
import { render } from "@testing-library/react";
import { describe, it, expect } from "vitest";
import App from "../App";

describe("App.tsx", () => {
  it("描画できること", () => {
    const { container } = render(<App />);
    expect(container.innerHTML).toMatchSnapshot();
  });
});
