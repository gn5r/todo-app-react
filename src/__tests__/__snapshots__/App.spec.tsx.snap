// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`App.tsx > 描画できること 1`] = `"<div><a href="https://vitejs.dev" target="_blank"><img src="/vite.svg" class="logo" alt="Vite logo"></a><a href="https://react.dev" target="_blank"><img src="/src/assets/react.svg" class="logo react" alt="React logo"></a></div><h1>Vite + React</h1><div class="card"><button>count is 0</button><p>Edit <code>src/App.tsx</code> and save to test HMR</p></div><p class="read-the-docs">Click on the Vite and React logos to learn more</p>"`;
